app/views/variables/show.html.haml

```diff
@@ -60,9 +56,12 @@
 
   %hr/
   %ul.nav
-    - if @variable.spec.present? && @variable.spec['references'].present?
-      - @variable.spec['references'].each do |ref|
-        %li= link_to t(:reference), ref, class: 'nav-link'
+    - if @variable.references.present?
+      - @variable.references.each do |ref|
+        -if (ref =~ /\A#{URI::regexp}\z/)
```

Comment:
@mischa-s Can we put this regex in a method?

Reply:
@dlouise64  do you have a suggestions for how to? @lamorrison and I had a conversation about this and because it a string inside the array we couldn't think of a good way to do this inside of a method

Reply:
:+1:  nice one, @dlouise64, done